src/react/utils/index.js

```diff
@@ -40,6 +40,10 @@ export function formatDate(d) {
   return `${d.toDateString()} ${d.toTimeString().split(' ')[0]}`;
 }
 
+export function formatRoleSessionName(oidc) {
```

Comment:
Can you please type this argument using https://github.com/scality/zenko-ui/blob/development/1.1/src/types/state.js#L39 ?